noty-android/app/composeapp/src/main/java/dev/shreyaspatil/noty/composeapp/ui/screens/NotesScreen.kt

```diff
@@ -121,7 +121,11 @@ fun NotesContent(
                     if (isConnectivityAvailable != null) {
                         ConnectivityStatus(isConnectivityAvailable)
                     }
-                    NotesList(notes) { note -> onNavigateToNoteDetail(note.id) }
+                    NotesList(notes.sortedByDescending { it.isPinned }) { note ->
```

Comment:
Sorting should not be part of UI logic. It would be nice if we do it from the DB layer itself.

Reply:
For this, we can change the DAO query. Otherwise, list sorting logic can hamper the UI performance.

Reply:
Right, I agree. I will make the changes.